src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/response/IdamErrorResponse.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.userprofileapi.controller.response;
+
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
```

Comment:
can't we make this immutable java class with builder? or getter setter needed?

Reply:
setter getters are required, so cant remove